### PR TITLE
[iOS][Bug] - fix crash on performance logger

### PR DESCRIPTION
**Pull request by zhigang1992**
## Summary

Fix Issue https://github.com/facebook/react-native/issues/24820

It's caused by `_labelsForTags` and `RCTPLTag` being out of sync, the crash might only be one of the issues that this bug was causing.

## Changelog

[iOS] [Fixed] - fix crash on performance logger

## Test Plan

- Launch iOS simulator
- Enable performance monitor
- Tap on performance monitor

It should not crash

### CODE REVIEW

File: React/Base/RCTPerformanceLogger.h [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 #import <Foundation/Foundation.h>
 
+// Keep this in sync with _labelsForTags
```
Comment (zhongwuzw) [inline React/Base/RCTPerformanceLogger.h]: I think we'd better to remove these missing labels in `_labelForTags`? because we don't use these enums at all.

Comment (PeteTheHeat) [reply React/Base/RCTPerformanceLogger.h]: Yup, +1

**Comment by cpojer** [comment]
@zhigang1992 could you make the change that was suggested by @zhongwuzw?

**Comment by zhigang1992** [comment]
All the unused flags are now removed except `RCTPLJSCWrapperOpenLibrary`. Which was added in https://github.com/facebook/react-native/commit/abe0b349bcdb59155445889d9615c13294a37c91 Should I remove that as well?

**Comment by zhongwuzw** [comment]
> All the unused flags are now removed except `RCTPLJSCWrapperOpenLibrary`. Which was added in [abe0b34](https://github.com/facebook/react-native/commit/abe0b349bcdb59155445889d9615c13294a37c91) Should I remove that as well?

We can keep it, don't break `RCTPLTag` enum.

**Pull request closed without merging**